api/v1alpha1/accesslogging_types.go

```diff
@@ -55,8 +55,33 @@ type ProxyAccessLogFormat struct {
 	// It's required when the format type is "JSON".
 	// +optional
 	JSON map[string]string `json:"json,omitempty"`
+	// Formatters defines a collection of [accesslog formatter](https://www.envoyproxy.io/docs/envoy/latest/api-v3/config/core/v3/substitution_format_string.proto#envoy-v3-api-field-config-core-v3-substitutionformatstring-formatters)
+	// that enable additional [command operators](https://www.envoyproxy.io/docs/envoy/latest/configuration/observability/access_log/usage#command-operators)
+	// that can be used in the accesslog format when accesslog sinks only contain "File".
+	// Valid AccessLogFormatterType values are
+	// "Cel"
+	// "Metadata"
+	// "ReqWithoutQuery"
+	// +optional
+	Formatters []AccessLogFormatterType `json:"formatters,omitempty"`
```

Comment:
didn't need an api for this, check https://github.com/istio/istio/blob/0ee031b7fe22d23fce0f75a4fb0c38be35a07215/pilot/pkg/model/telemetry_logging.go#L317

Reply:
That's a good idea to add formatters based on whether specified operator command shows in text automatically. I'll reimplement this feature in the same way.